mergebom.py

```diff
@@ -24,24 +24,29 @@
 import argparse
 import ConfigParser
 import re
-from lib import cfg
+from lib import cfg,lib,report
 from mergebom_class import *
 from datetime import datetime
    
 if __name__ == "__main__":
+    rep = report.Report(log_on_file=True, terminal=True, report_date=None)
 
     file_list = []
     parser = argparse.ArgumentParser()
     parser.add_argument('--workspace-file', '-w', dest='ws', 
                         help='Dove si trova il file WorkSpace', default=None)
+    parser.add_argument('--nome-file.xlsx', '-nw', dest='nw', 
+                        help='Nome file da mergiare', default='bom-')
+    parser.add_argument('--caratteristiche-file', '-cf', dest='cf', action="store_true",
+                        help='Se il file deve avere lo stesso nome e la stessa directory del file vecchio', default=False)
     parser.add_argument("-a", "--csv", dest="csv_file", action="store_true",
                       default=False, help="Find and merge csv files, by defaul are excel files.")
     parser.add_argument("-c", "--merge-cfg", dest="merge_cfg",
                       default=None, help="MergeBOM configuration file.")
     parser.add_argument("-o", "--out-filename", dest="out_filename",
-                      default='merged_bom.xlsx', help="Out file name")
+                      default='merged_bom', help="Out file name")
```

Comment:
Manca l'estensione..

Reply:
il nome del file deve essere passato a linea di comando senza estenzione, in modo che io possa modificare il nome in caso che il file precedente non deve essere modificato